lib/screens/wallet/components/tokens_cards/interactor/mappers/token_balances_state_mapper.dart

```diff
@@ -43,6 +45,14 @@ class TokenBalancesStateMapper {
       }
     }
 
+    // load colors
+    final repo = ColorPaletteRepository();
+    for (final TokenBalanceViewModel viewModel in available) {
+      if (viewModel.token != SeedsToken) {
+        viewModel.dominantColor = await repo.getImagePaletteCached(viewModel.token.backgroundImage);
+      }
```

Comment:
So the color derives from the background image, cool.

Reply:
Yes!